styles.css

```diff
@@ -47,54 +49,65 @@ nav ul li a {
 }
 
 nav ul li a:hover {
-        background-color: #47778E;
+    background-color: #47778E;
     }
 
 /* ------------------- homepage ----------------- */
 
+/* display of h1 set to inline block so that its red border won't take full width of page */
+
 h1 {
-    font-size: 3rem;
-    line-height: 4.5rem;
-    padding-top: 30vh;
+    position: relative;
+    top: 30vh;
+    font-size: 6.1rem;
+    line-height: 8.5rem;
+    display: inline-block;
+    border-bottom: 6px solid #DB4D44;
+    padding-bottom: 25vh;
 }
 
-.part1-main-title, .emphasize-rosa {
-    font-size: 6rem;
-    line-height: 8rem;
-    border-bottom: 6px solid #DB4D44;
-    padding-bottom: 40vh;
+.subtitle {
+    display: block;
+    font-size: 2.4rem;
+    line-height: 3.5rem;
 }
 
+.soon-to-be {
+    font-size: 1.6rem;
+}
```

Comment:
Not clear to me what this is for. The class name doesn't give me a clue to what it does on the page.